doc/releases/release-notes-2.4.rst

```diff
@@ -409,6 +416,11 @@ Networking
 * Fixed LWM2M Security and Server object instance matching.
 * Fixed LWM2M updating lifetime on Register Update event.
 * Fixed MQTT double CONNACK event notification on server reject.
+* PPP: Added initial support for PAP authentication.
```

Comment:
Please add this to "Added" section above after the other PPP option. So something like this
```
* Added support to rejecting received and unsupported PPP options.
* Added initial support for PAP authentication.
```